hudhook/src/hooks/common.rs

```diff
@@ -56,6 +58,7 @@ pub(crate) fn imgui_wnd_proc_impl(
     WPARAM(wparam): WPARAM,
     LPARAM(lparam): LPARAM,
     mut imgui_renderer: MutexGuard<Box<impl ImguiWindowsEventHandler>>,
+    imgui_render_loop: &Box<dyn ImguiRenderLoop + Send + Sync>
```

Comment:
We have to drop the renderer so it's ok to use the specialized mutex guard type, but here I'd go with a more generic `T: AsRef<ImguiRenderLoop + Send + Sync>` or something equivalent.